code/modules/goonchat/browserassets/js/browserOutput.js

```diff
@@ -24,8 +24,10 @@ window.status = 'Output';
 var $messages, $subTheme, $subOptions, $subFont, $selectedSub, $contextMenu, $filterMessages, $last_message;
 var opts = {
 	//General
-	'messageCount': 0, //A count...of messages...
-	'messageLimit': 2053, //A limit...for the messages...
+	'messageCount': 0, //A count of messages
+	'messageLimit': 2053, //A limit for the messages
+	'messageLimitMin': 2053,
```

Comment:
```suggestion
	'messageLimitMin': 32,
```
in case anyone wants to, i guess.

Reply:
There is no reason to go that low. If anything, it is a noob trap for inattentive people to set up a super low limit and then go in OOC "wtf my chat broken why I only get N messages". Minimal value should be same as current value (or close to it), so the default unchanged behavior is the same as current.

Reply:
Changed to 2048 in most recent commit.